src/features/gym/GymPopup.jsx

```diff
@@ -116,6 +143,222 @@ export function GymPopup({ hasRaid, hasHatched, raidIconUrl, ...gym }) {
   )
 }
 
+/**
+ * Compact modal for gym defenders
+ * @param {{ gym: import('@rm/types').Gym, onClose: () => void }} param0
+ */
+function DefendersModal({ gym, onClose }) {
+  const { t } = useTranslation()
+  const Icons = useMemory((s) => s.Icons)
+  const defenders = gym.defenders || []
+
+  return (
+    <Grid
+      container
+      direction="column"
+      alignItems="stretch"
+      mt={-1}
+      style={{ minWidth: 250, maxWidth: 350, padding: 8 }}
+    >
+      <Grid container alignItems="center" mb={1}>
+        <Grid xs={2}>
+          <IconButton
+            onClick={(e) => {
+              e.stopPropagation()
+              onClose()
+            }}
+            size="small"
+          >
+            <ArrowBackIosNewIcon fontSize="small" />
+          </IconButton>
+        </Grid>
+        <Grid
+          xs={10}
+          style={{
+            overflow: 'hidden',
+            textOverflow: 'ellipsis',
+            wordBreak: 'break-word',
+            maxWidth: 200,
+            display: 'flex',
+            alignItems: 'center',
+          }}
+        >
+          <Title backup={t('unknown_gym')}>{gym.name}</Title>
+        </Grid>
+      </Grid>
+      <Grid container direction="column" spacing={1}>
+        {defenders.map((def) => {
+          const fullCP = def.cp_when_deployed
+          const currentCP = def.cp_now
+          const percent = Math.max(
+            0,
+            Math.min(1, (currentCP / fullCP) * 1.25 - 0.25),
```

Comment:
Also it is better to use `def.motivation_now` instead of `currentCP / fullCP`. It gets 1/10000 precision rather than at most 1/4008 (for maxed Slaking).